be/tool/beck.c

```diff
@@ -534,6 +543,13 @@ int main(int argc, char **argv)
 	if (dry_run)
 		printf("Running in read-only mode.\n");
 
+	if (s.s_print_invalid_oids)
+		printf("Will print INVALID GOB IDs when found during scanning.\n");
+	else
+		printf("Will not print INVALID GOB IDs if found during "
+		       "scanning. To print INVALID GOBs please pass "
+		       "-e option.\n");
+
```

Comment:
Do we need this? As we have already -h option

Reply:
Made some changes here. We still need to warn the User about 'not printing invalid GOBs if -e is not provided' since he may forget it and keep the program running hoping that invalid GOBs will be shown on the screen.